Classes/DDFileLogger.m

```diff
@@ -869,6 +874,32 @@ - (void)maybeRollLogFileDueToSize {
     }
 }
 
+- (void)maybeRollLogFileDueToDate {
+	if (_rollEveryDay && _currentLogFileInfo && ![self isLogFileCreatedToday:_currentLogFileInfo]) {
+		NSLogVerbose(@"DDFileLogger: Rolling log file due to date...");
+		
+		[self rollLogFileNow];
+	}
+}
+
+- (BOOL)isLogFileCreatedToday:(DDLogFileInfo *)aLogFile {
+	NSDate *logFileCreationDate = aLogFile.creationDate;
+	NSDate *today = [NSDate date];
```

Comment:
Yes, it depends. Method comparing dates according to **current** calendar, see screenshot. 
<img width="950" alt="screen shot 2016-05-23 at 21 45 31" src="https://cloud.githubusercontent.com/assets/873387/15480775/e839aa64-212f-11e6-8513-3133f4670806.png">